NtupleProducer/plugins/NtupleProducer.cc

```diff
@@ -359,10 +360,11 @@ NtupleProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
             return c.et; 
         }
   } );
+  */
   //// ---- Calibration from Phil's workflow
-  //ecalClusterer_.correct( [](const l1pf_calo::Cluster &c, int ieta, int iphi) -> double { 
-  //      return ecorrector_->correct(0., c.et, ieta, iphi); // FIXME is ieta or abs(ieta) ?
-  //} );
+  ecalClusterer_.correct( [=](const l1pf_calo::Cluster &c, int ieta, int iphi) -> double { 
+      return ecorrector_->correct(0., c.et, ieta, iphi); // FIXME is ieta or abs(ieta) ? => its ieta
```

Comment:
now that you changed the first argument to `iTotal` I suppose the zero should be `c.et` ?
testing now (otherwise, I get crashes in the corrector due to nonsensical H/E)